JavaDirectory/src/main/java/com/bug/free/invention/api/Services/CapabilityService.java

```diff
@@ -0,0 +1,45 @@
+package com.bug.free.invention.api.Services;
+
+import com.bug.free.invention.api.Models.Employee;
+import com.bug.free.invention.api.Models.capability;
+import com.bug.free.invention.api.Models.job;
+import com.bug.free.invention.api.controllers.DBConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Component
+public class CapabilityService {
+    @Autowired
+    private CapabilityRepository repository;
+    private boolean isPopulated = false;
+    public CapabilityService(CapabilityRepository repository) {
+        this.repository = repository;
+    }
+
+    public void populateCapability(){
+        String dbQuery = "SELECT * FROM `capability`";
+        try(Connection DatabaseConnection = DBConfig.getConnection()){
```

Comment:
```suggestion
        try(Connection databaseConnection = DBConfig.getConnection()){
```
Generally variables should have lowercase names

Reply:
done